object/service.proto

```diff
@@ -42,6 +52,13 @@ service ObjectService {
   // object payload. All messages, except first one, SHOULD be payload chunks.
   // Chunk messages SHOULD be sent in direct order of fragmentation.
   //
+  // Extended headers can change `Put` behaviour:
+  // * __NEOFS__NETMAP_EPOCH \
```

Comment:
I don't think this is useful for `Put`. I can't imagine the scenario when somebody wants to store an object in the "past" and storage nodes accepts it.

Same for `Delete`.

Reply:
This functionality is required to address cases with some network segments losing full connectivity. Even if it's not used now by our implementation, the protocol itself must support operation in an Epoch specified.

Reply:
I agree that we can parameterize this value in binary protocol, but I still can't imagine the UX in setting the particular values. Regardless of the implementation, how are storage nodes that are no longer present in the container motivated to serve these kinds of requests?